pkg/proc/breakpoints.go

```diff
@@ -925,3 +941,36 @@ func returnInfoError(descr string, err error, mem MemoryReadWriter) []*Variable
 	v.Name = "return value read error"
 	return []*Variable{v}
 }
+
+// LogicalBreakpoint represents a breakpoint set by a user.
```

Comment:
Can you add some more documentation here around what a logical breakpoint represents (in terms of multiple physical, etc...)